### PR TITLE
VideoCommon: fix deterministic dual core

### PR DESCRIPTION
**Pull request by degasus**

### CODE REVIEW

**Comment by comex** [comment]
Huh.  I'm confused.  Thought I dealt with this when I wrote it.

**Comment by JMC47** [comment]
@comex Virtual Vertex Loader was merged after deterministic dualcore

**Comment by comex** [comment]
So it's not my fault? ;)

**Comment by JMC47** [comment]
Nope, not this time.  Not yet at least.  Btw, tested this and it fixes the problem.

File: Source/Core/VideoCommon/OpcodeDecoding.cpp [hunk 1]
```diff
@@ -268,7 +268,7 @@ u8* OpcodeDecoder_Run(DataReader src, u32* cycles, bool in_display_list)
 
 				if (is_preprocess)
 				{
-					size_t size = num_vertices * VertexLoaderManager::GetVertexSize(cmd_byte & GX_VAT_MASK, is_preprocess);
+					size_t size = num_vertices * VertexLoaderManager::GetVertexSize(cmd_byte & GX_VAT_MASK, true);
```
Comment (BhaaLseN) [inline Source/Core/VideoCommon/OpcodeDecoding.cpp]: ```is_preprocess``` is a bool anyways, you could have left that one.

**Comment by degasus** [comment]
Any comments?

**Pull request merged**